distributed/ddp-tutorial-series/slurm/setup_pcluster_slurm.md

```diff
@@ -0,0 +1,51 @@
+# Setup AWS cluster with pcluster
+
+## 1. Sign in to an AWS instance
+
+## 2. Install pcluster
+```
+pip3 install awscli -U --user
+pip3 install "aws-parallelcluster" --upgrade --user
+```
+
+## 3. Create a cluster config file
+```
+pcluster configure --config config.yaml
+```
+See config.yaml.template for an example
+
+
+## 4. Create the cluster
+```
+pcluster create-cluster --cluster-name dist-ml --cluster-configuration config.yaml
+```
+
+### 4a. Track progress
+```
+pcluster list-clusters
+```
+
+## 5. Login to cluster headnode
+```
+pcluster ssh --cluster-name dist-ml -i your-keyname-file
+```
+
+## 6. Install dependencies
+```
+sudo apt-get update
+sudo apt-get install -y python3-venv
+python3.7 -m venv /shared/venv/
+source /shared/venv/bin/activate
+pip install wheel
+echo 'source /shared/venv/bin/activate' >> ~/.bashrc
+```
+
+## 7. Download training code and install requirements
+```
+cd /shared
+git clone https://github.com/suraj813/distributed-pytorch.git
```

Comment:
Is this still the repo we want to link to?

Reply:
oops